test/unit/org/apache/cassandra/index/CustomIndexTest.java

```diff
@@ -606,6 +610,35 @@ public void notifyIndexersOfExpiredRowsDuringCompaction() throws Throwable
         assertEquals(0, deletedClustering.intValue());
     }
 
+    @Test
+    public void notifyIndexesOfFullyExpiredSSTablesDuringCompaction() throws Throwable
+    {
+        createTable("CREATE TABLE %s (id int primary key, col int) " +
+                    "WITH compaction = {'class': 'TimeWindowCompactionStrategy', " +
+                    "                   'compaction_window_size': 1," +
+                    "                   'compaction_window_unit': 'MINUTES'," +
+                    "                   'expired_sstable_check_frequency_seconds': 10} " +
+                    "AND gc_grace_seconds = 0");
+
+        createIndex(String.format("CREATE CUSTOM INDEX row_ttl_test_index ON %%s(col) USING '%s'", StubIndex.class.getName()));
```

Comment:
we can go with just this index as it is true in `notifyIndexerAboutRowsInFullyExpiredSSTables` by default